<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cf</class>
 <widget class="QDialog" name="Cf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="select">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selectati din lista agentia pentru care doriti sa se afiseze lista de clienti/furnizori:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="agentieBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>341</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="showc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga client nou</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga furnizor nou</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inchide">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inchide</string>
   </property>
  </widget>
  <widget class="QLabel" name="newcf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selectati din lista agentia pentru care doriti sa se afiseze lista de clienti/furnizori:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nume client/furnizor:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cfEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>91</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="aferent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agentia aferenta:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="aferentBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anuleaza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>440</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anuleaza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Afiseaza clienti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sf">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Afiseaza furnizori</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
